src/main/java/favourite/interface_adapters/FavouriteController.java

```diff
@@ -0,0 +1,32 @@
+package favourite.interface_adapters;
+
+import favourite.use_case.FavouriteInputBoundary;
+import favourite.use_case.FavouriteRequestModel;
+
+public class FavouriteController{
+    private final FavouriteInputBoundary interactor;
+    private final int postID;
+
+    /**
+     * Initializes FavouriteController
+     * @param interactor the interactor, passing as a FavouriteInputBoundary
+     * @param postID the id of the post being favourtied/unfavourited
+     */
+    public FavouriteController (FavouriteInputBoundary interactor, int postID){
```

Comment:
Does the controller need to have postID in the constructor?

Reply:
Good point, fixed